docs/kopia_troubleshooting.md

```diff
@@ -57,23 +62,25 @@ kopia repository connect s3 \
     --prefix="$S3_REPOSITORY_PATH" \
     --access-key="$S3_ACCESS_KEY" \
     --secret-access-key="$S3_SECRET_ACCESS_KEY" \
+    # --disable-tls-verification \ # optional
```

Comment:
Could you please also add the commented out:

export S3_ENDPOINT=<IP address or URL to the S3 repository>
    --endpoint="$S3_ENDPOINT" \


The above was the case when I was helping one customer and we had to include that in the debug command, so I think it's worth mentioning.

Reply:
sure :)  coming up